src/app/components/authorized/feed/feed.component.scss

```diff
@@ -1,39 +1,15 @@
 @import '../../../../colors.scss';
 @import '../../../../mixins.scss';
 
-.contentWrapper {
-    display: grid;
-    margin: 2rem 0;
-    column-gap: 5rem;
-    align-items: start;
-}
-
 .postsWrapper {
     display: grid;
-    row-gap: 1rem;
-
-    > h1 {
-        background-color: $darkBlue;
-        color: $lightTextColor;
-        padding: 0.5rem;
-        letter-spacing: 0.1rem;
-        @include boxshadow;
-        border-left: 0.5rem solid $secondaryColor;
-        border-radius: 0.3em;
-    }
-
+    row-gap: 1rem; 
+    
     > div {
-        display: grid;
-        color: $basicTextColor;
-        background-color: white;
-        @include boxshadow;
-        border-left: 1rem solid $darkBlue;
-        border-radius: 0.3em;
-        padding: 1rem;
-
         &:hover {
-            background-color: #e2f3ff;
+            @include whiteBackground;
+            background-color: rgb(255, 255, 255, 0.2);
```

Comment:
What does `rgb(a,b,c,d)` do? Didn't you mean to have it `rgba`? I've used fiddle and it does seem to work the same, but it looks as undocumented feature. Some pre-processors don't seem to compile it (jsfiddle doesn't support it for example).